.github/docs/openapi.yml

```diff
@@ -538,6 +538,61 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/FailResponse"
+  /loomies/exists: 
+    post: 
+      tags: [ Loomies ]
+      description: Validate if Loomie still exists.
+      security: 
+        - basicAuth: [Access-Token]
+      requestBody: 
+        content: 
+          application/json: 
+            schema: 
+              type: object
+              properties:
+                loomie_id: 
+                  type: string
+                  example: "641e1562d928b0e05a639862"
+        required: true
+      responses: 
+        "200": 
+          description: There wasn't any error and the loomie exists.
+          content: 
+            application/json: 
+              schema:
+                type: object
+                properties:
+                  error:
+                      type: boolean
+                      example: false
+                  message: 
+                      type: string
+                      example: "Loomie exists"
+        "400":
+          description: Bad request. Mayby the loomie id field are empty. 
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/FailResponse"
+        "401":
+          description: The access token wasn't provided or isn't valid.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/FailResponse"
+        "404":
+          description: Loomie not exists in collection wild_loomies
```

Comment:
Not exists -> Doesn't exists
Collection wild_loomies -> wild_loomies collection.